delly_docker/scripts/run_seqware_workflow.pl

```diff
@@ -122,7 +122,12 @@
 chomp $path;
 
 # MOVE THESE TO THE RIGHT PLACE
-system("gosu root mv /datastore/$path/*.vcf.gz /datastore/$path/*.bedpe.txt /datastore/$path/delly_results/*.sv.cov.tar.gz /datastore/$path/delly_results/*.sv.cov.plots.tar.gz /datastore/$path/*.sv.log.tar.gz /datastore/$path/*.json $output_dir");
+system("gosu root mv /datastore/$path/*.vcf.gz* /datastore/$path/*.bedpe.txt* /datastore/$path/delly_results/*.sv.cov.tar.gz* /datastore/$path/delly_results/*.sv.cov.plots.tar.gz* /datastore/$path/*.sv.log.tar.gz* /datastore/$path/*.json /datastore/$path/*.sv.readname.txt.tar.gz* $output_dir");
+# tar gzip the timing_metrics.json and qc_metrics.json
+system("gosu root tar -zcvf $output_dir/sv.timing_metrics.tar.gz -C $output_dir *.sv.timing.json");
```

Comment:
Can these targz files follow the same naming patten as existing output files, for example: `OCCAMS-ST-020.embl-delly_1-3-0-preFilter.20150318.germline.sv.vcf.gz`?